src/JSInterop/Microsoft.JSInterop/src/JSInProcessRuntime.cs

```diff
@@ -35,7 +35,8 @@ public TValue Invoke<TValue>(string identifier, params object[] args)
         /// </summary>
         /// <param name="identifier">The identifier for the function to invoke.</param>
         /// <param name="argsJson">A JSON representation of the arguments.</param>
+        /// <param name="treatReturnAsVoid">Ignore any result that is returned and treat as Void.</param>
         /// <returns>A JSON representation of the result.</returns>
-        protected abstract string? InvokeJS(string identifier, string? argsJson);
+        protected abstract string? InvokeJS(string identifier, string? argsJson, bool? treatReturnAsVoid);
```

Comment:
@SteveSandersonMS - It's not clear to me where this method is implemented?  Not sure what needs to be updated to ensure that this new `treatReturnAsVoid` parameter gets passed to the JavaScript side of things.

Reply:
Sorry... found it.  I initially was just opening the JSInterop.csproj on it's own.... just found Components.slnf which contains everything... now if I can only get the solution to build :(.  Some build tasks appear to be failing: `Microsoft.AspNetCore.Razor.Tasks.RazorGenerate`... any thoughts?  It looks like the some dlls in the `artifacts` folder are not being created.